### PR TITLE
Add more info about doctesting things with rand

### PR DESCRIPTION
**Pull request by kshyatt**
Since I just figured this out, might be useful to others.

### CODE REVIEW

File: doc/src/manual/documentation.md [hunk 1]
```diff
@@ -107,7 +107,10 @@ As in the example above, we recommend following some simple conventions when wri
 
    !!! warning
        Calling `rand` and other RNG-related functions should be avoided in doctests since they will not
-       produce consistent outputs during different Julia sessions.
+       produce consistent outputs during different Julia sessions. If you would like to show some random
+       number generation related functionality, one option is to explicitly construct and seed your own
+       [`MersenneTwister`] (or other pseudorandom number generator) and pass it to the functions you are
```
Comment (KristofferC) [inline doc/src/manual/documentation.md]: should `MersenneTwister` be reffed?

Comment (kshyatt) [inline doc/src/manual/documentation.md]: I thought I did! Grrrr

**Comment by kshyatt** [comment]
@KristofferC better?

**Review by KristofferC — APPROVED** [review]
Another thing that would be nice to mention is meta blocks (can be used to `srand` before every doctest in a markdown file): https://juliadocs.github.io/Documenter.jl/latest/man/syntax/#@meta-block-1 but that can be done in another PR.

**Review by Sacha0 — APPROVED** [review]
Nice addition! :)

**Pull request merged**